src/Services/Air/validators/reservation-locator.js

```diff
@@ -0,0 +1,8 @@
+import { AirValidationError } from '../AirErrors';
+
+export default (params) => {
+  if (params.uapi_reservation_locator === undefined
```

Comment:
Aha, that's not token, it's a string from uapi.
Yes, just check for a string here.